release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -304,10 +304,17 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-no
 [id="ocp-414-broadcom-bcm57504-support"]
 ==== Support for Broadcom BCM57504 is now GA
 
-Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator. 
+Support for the Broadcom BCM57504 network interface controller is now available for the SR-IOV Network Operator.
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-14-admin-network-policy"]
+==== Admin Network Policy (Technology Preview)
+
+Admin Network Policy is available as a Technology Preview feature.
+
+You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
```

Comment:
```suggestion
You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plug-in. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

Reply:
@tmalove In this case, the IBM style guide is superseded by Red Hat style: https://redhat-documentation.github.io/supplementary-style-guide/#plugin